src/main/java/org/sagebionetworks/web/client/presenter/EntityPresenter.java

```diff
@@ -163,8 +169,9 @@ public void onFailure(Throwable caught) {
 					view.show404();

```

Comment:
Do a synAlert.clear();  as one of the first lines in refresh() (before async call).